packages/client/src/features/accessibility/view-key-tools/grid-zoom-key-tool.ts

```diff
@@ -34,23 +34,23 @@ import { EditorContextService } from '../../../base/editor-context-service';
 import { SelectionService } from '../../../base/selection-service';
 import { Tool } from '../../../base/tool-manager/tool';
 import { getAbsolutePositionByPoint } from '../../../utils/viewpoint-util';
+import { ZoomFactor } from '../../viewport/utils';
 import { ElementNavigatorKeyListener } from '../element-navigation/diagram-navigation-tool';
 import { SetAccessibleKeyShortcutAction } from '../key-shortcut/accessible-key-shortcut';
 import { EnableKeyboardGridAction, KeyboardGridCellSelectedAction, KeyboardGridKeyboardEventAction } from '../keyboard-grid/action';
-import { ZoomElementAction, ZoomViewportAction } from '../move-zoom/zoom-handler';
 import * as messages from '../toast/messages.json';
 import { HideToastAction, ShowToastMessageAction } from '../toast/toast-handler';
 
 /**
  * Zoom viewport and elements when its focused and arrow keys are hit.
  */
 @injectable()
-export class ZoomKeyTool implements Tool {
-    static ID = 'glsp.zoom-key-tool';
+export class GridZoomTool implements Tool {
```

Comment:
Maybe we rename this use `GridCellZoomTool` to avoid potential confusion with the `Grid` API used for snapping?